webpack.config.js

```diff
@@ -130,7 +130,13 @@ module.exports = function (env) {
           use: ExtractTextPlugin.extract({
             fallback: 'style-loader',
             use: [
-              'css-loader',
+              {
+                loader: 'css-loader',
+                options: {
+                  modules: true,
+                  localIdentName: '[name]__[local]--[hash:base64:5]'
```

Comment:
You want to have separate prod/dev ident names:
* `[hash:base64:5]` is good for prod. Hashes are slow so best not used in development
* `[name]__[local]` is ok for dev. I personally like `[path][name]-[local]` becase we also know what file the class name came from. Also, when all css files have a similar name (e.g. `style.scss`) then `[name]` is not worth much, and just `[local]` is very likely to introduce conflicts